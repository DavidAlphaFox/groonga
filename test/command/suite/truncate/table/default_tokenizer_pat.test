table_create Tags TABLE_PAT_KEY ShortText \
  --default_tokenizer 'TokenNgram("n", 3)'
dump

truncate Tags
dump
