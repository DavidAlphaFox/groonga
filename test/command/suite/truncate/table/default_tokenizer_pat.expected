table_create Tags TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("n", 3)'
[[0,0.0,0.0],true]
dump
table_create Tags TABLE_PAT_KEY ShortText --default_tokenizer "TokenNgram(\"n\", 3)"
truncate Tags
[[0,0.0,0.0],true]
dump
table_create Tags TABLE_PAT_KEY ShortText --default_tokenizer "TokenNgram(\"n\", 3)"
